main.cpp

```diff
@@ -8,12 +8,38 @@
 #include <GL/glew.h>
 #include <GLFW/glfw3.h>
 #include <glm/glm.hpp>
+#include <nlohmann/json.hpp>
 
+#include <fstream>
 #include <iostream>
 #include <string>
 
 using namespace pong;
 
+nlohmann::json OpenJsonFile(const std::string& filePath)
+{
+    std::ifstream jsonFile(filePath);
+    if (!jsonFile.is_open())
+    {
+        std::cerr << "Failed to open JSON file: " << filePath << std::endl;
+        return nullptr;
+    }
+
+    nlohmann::json jsonData;
+
+    try
+    {
+        jsonFile >> jsonData;
+    }
+    catch (const nlohmann::json::parse_error& e)
```

Comment:
EXCEPTIONS??????????????????? 🤯

Reply:
Its only been a month! I have officially lost it 🧠